src/modules/Auth/Register/RegisterPage.tsx

```diff
@@ -1,7 +1,75 @@
 import React from "react";
+import { Formik } from "formik";
+import Modal from "antd/lib/modal/Modal";
+import { Typography, Space, Button } from "antd";
+import { Link } from "react-router-dom";
+import { Form, TextField } from "../../../components/FormikWrapper";
+import * as yup from "yup";
 
 const RegisterPage = () => {
-  return <div>register page</div>;
+  const validationSchema = React.useMemo(() => {
+    return yup.object({
+      email: yup.string().required(),
+      password: yup.string().required(),
+      password_confirmation: yup.string().required(),
```

Comment:
validation buat `password_confirmation` ini bisa diimprove dengan membuatnya harus sama seperti `password`. Coba kamu lihat referensi ini https://til.hashrocket.com/posts/vahuw4phan-check-the-password-confirmation-with-yup

```suggestion
      password_confirmation: yup.string()
     .oneOf([yup.ref('password'), null], 'Passwords must match'),
```